plugin.xml

```diff
@@ -36,11 +36,18 @@
 
     <!-- android -->
     <platform name="android">
+        <framework src="build-extras.gradle" custom="true" type="gradleReference" />
```

Comment:
Please move the `build-extras.gradle` file into `src/android`, as this is only relevant for Android.